js/modal.js

```diff
@@ -0,0 +1,134 @@
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'multi-post1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'tonic2.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    company: 'Canopy',
+    position: 'Back End Dev',
+    year: '2022',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredImage: 'multi-post2.png',
+    technologies: ['html', 'css', 'javaScript'],
+    liveVersion: 'https://eugeneiyukhun.github.io/My-Portfolio-Desktop-Version/',
+    source: 'https://github.com/Eugeneiyukhun/My-Portfolio-Desktop-Version',
+  },
+];
+
+projects.forEach((project) => {
+  const article = document.createElement('article');
+  article.classList.add('work-card');
+  function concatenateTechnologies(acc, cur) {
+    acc += `<li class="tag">${cur}</li>`;
+    return acc;
+  }
+
+  const technologies = project.technologies.reduce(concatenateTechnologies, '');
+  const buttonId = `see-project-${project.id}`;
+  article.innerHTML = `
+  <div class="work-card_picture">
+    <img src="img/Posts/${project.featuredImage}" alt="Tonic" />
+  </div>
+  <div class="work-card_elements">
+    <h3 class="work-card_title">${project.name}</h3>
+    <div class="work-card_specs">
+      <div class="specs_client">${project.company}</div>
+      <div class="dot"></div>
+      <div class="specs_role">${project.position}</div>
+      <div class="dot"></div>
+      <div class="specs_year">${project.year}</div>
+    </div>
+    <p class="work-card_description">${project.shortDescription}</p>
+    <ul class="work-card_techs">
+      ${technologies}
+    </ul>
+    <button class="work-card_button" id="${buttonId}">See Project</button>
+  </div>
+  `;
```

Comment:
- [ ] Everything is good in the mobile version, there is a small change you need to make on the mobile version. Please adjust the spacing between list items and buttons. See the image below for reference
### Your design
![his](https://user-images.githubusercontent.com/81727988/210963723-70728fae-dc68-49f0-86c5-c680e817d491.png)

### Fimga design
![fimaa](https://user-images.githubusercontent.com/81727988/210964077-9fbfc970-15b7-490d-a883-169ebedb4d44.png)